src/installation/guides/macbook.md

```diff
@@ -0,0 +1,53 @@
+# Macbookpro 12,1 (early 2015) 
+
+Everything is well supported on this laptop and install will work without problems. Post-installation, after having booted to your newly installed system, there is however a few points that need a little bit of work.
+
+## keymap are wrong
+This is a problem inherited by a few distros, including archlinux.
+tty keymaps for the macbooks (mac-fr, mac-us, mac-de) are completely broken and *you should not attempt to use them*, or it will result in an unusable keyboard and you will be forced to do a hard reset on your system. as a workaround generic keymaps
```

Comment:
This could be condensed to "The `mac-` prefixed keymaps may not work and using them may require you to replace them later." I'm not sure what you mean by "hard reset" here.